python/packages/ai/teams/app.py

```diff
@@ -609,7 +610,7 @@ async def __handler__(context: TurnContext, state: StateT):
                 if not context.activity.value:
                     return False
 
-                feedback = context.activity.value
+                feedback = FeedbackLoopData.from_dict(context.activity.value)
                 feedback.reply_to_id = context.activity.reply_to_id
```

Comment:
```suggestion
                feedback.reply_to_id = context.activity.replyToId
```

Reply:
@lilyydu not sure if it's right though, it seems the code suggestions in VS Code suggests `activity.reply_to_id` instead of `activity.replyToId`. Possibly some conversion happens in the code to turn it from camel case to snake case.

Reply:
hmm, I will take a look today